### PR TITLE
Remove stray printf.

### PR DESCRIPTION
**Pull request by jsha**
Also includes some small gofmt changes.

### CODE REVIEW

**Comment by lziest** [comment]
lgtm.

**Comment by kisom** [comment]
LGTM.

**Comment by kisom** [comment]
Can you remove the "fmt" import?

File: signer/local/local.go [hunk 1]
```diff
@@ -266,7 +266,6 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 
 	if profile.ClientProvidesSerialNumbers {
 		if req.Serial == nil {
-			fmt.Printf("xx %#v\n", profile)
```
Comment (lziest) [inline signer/local/local.go]: need to remove 'fmt' import as test suggests

**Comment by jsha** [comment]
Done.

**Pull request merged**